test/ffmpeg-vaapi/transcode/transcoder.py

```diff
@@ -10,35 +10,132 @@
 
 @slash.requires(have_ffmpeg)
 @slash.requires(have_ffmpeg_vaapi_accel)
+@platform_tags(ALL_PLATFORMS)
 class TranscoderTest(slash.Test):
+  requirements = dict(
+    decode = {
+      "avc" : dict(
+        sw = (ALL_PLATFORMS, (True, True), None),
+        hw = (AVC_DECODE_PLATFORMS, (True, True), None),
+      ),
+      "hevc-8" : dict(
+        sw = (ALL_PLATFORMS, (True, True), None),
```

Comment:
We need to specify the SW decoder here (similar to ffmpeg-qsv transcoder.py)

Reply:
added SW decoder validation check

Reply:
same as above

Reply:
added the decoder